plugins/modules/ssh_config.py

```diff
@@ -187,6 +180,8 @@ def __init__(self, module):
         self.config_file = self.params.get('ssh_config_file')
         self.identity_file = self.params['identity_file']
         self.check_ssh_config_path()
+        if not HAS_PARAMIKO:
+            module.fail_json(msg=missing_required_lib('PARAMIKO'), exception=PARAMIKO_IMPORT_ERROR)
```

Comment:
I would move this to the top (before calling any other function, like `check_ssh_config_path()`). Right now `check_ssh_config_path()` does not use anything from StormSSH, but eventually someone might modify the function to use it, and then suddenly we are in a situation where this check is done too late :)